.github/workflows/ci.yml

```diff
@@ -215,6 +215,31 @@ jobs:
     - name: Distcheck
       run: make distcheck DISTCHECK_CONFIGURE_FLAGS="--enable-werror"
 
+  build-macos-latest-pcp:
+    runs-on: macOS-latest
+    env:
+      CC: clang
+      # '-mmacos-version-min=15.4' is a workaround for PCP 7.0.2 that
+      # accidentally broke backward compatibility.
+      # (https://github.com/performancecopilot/pcp/pull/2400)
+      CFLAGS: -O3 -g -flto -mmacos-version-min=15.4
+      LDFLAGS: -O3 -g -flto
+    steps:
+    - uses: actions/checkout@v5
+    - name: Install Dependencies
+      run: |
+        pcp_version=7.0.2-1
```

Comment:
@Explorer09 this is already out-of-date :( - and PCP releases quite often, usually once every couple of months or less.  I wonder if there's any way we can find latest release from github?